### DISCUSSION TITLE
Footer stop working after updating BIRT from 4.8 to 4.15/4.16

### DISCUSSION THREAD

**Discussion by tameergh1** [eclipse-birt/birt#1781] [General]
Hi, after updating BIRT version, the footer stopped working!!

**Comment by hvbtup** [comment]
I hope your users give you better error descriptions...

**Reply by tameergh1** [reply]
Sorry, 
It is there but the footer size have been changed, without making any changes in the rpttemplate file

**Comment by hvbtup** [comment]
Please create a self-contained example demonstrating the issue.

**Reply by tameergh1** [reply]
Adding an image to the footer of a BIRT report. it gets cropped rather than resized.
Lets say we add an image 3500px X 1200px, it will be cropped to specific size
(smaller than the origin on ) rather than resized

**Reply by hvbtup** [reply]
Which emitter? What are the image properties, in particular FitToContainer? I'd recommend that you explicitly set the image width and height in an absolute unit like `pt` or `mm` (not `px`!)  as a remedy.
